curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/convert-json-data-to-html.english.md

```diff
@@ -24,6 +24,7 @@ json.forEach(function(val) {
 });
 ```
 
+<b>Note:</b> For this challenge you need to add new html elements to the page, so you cannot rely on textContent. Instead you need to use innerHTML which can make a site vulnerable to cross site scripting attacks.
```

Comment:
Something I did not catch in the first review, but according to our guidelines, notes are surrounded by strong tags.
```suggestion
<strong>Note:</strong> For this challenge you need to add new html elements to the page, so you cannot rely on textContent. Instead you need to use innerHTML which can make a site vulnerable to cross site scripting attacks.
```